app/assets/stylesheets/teams/show.scss

```diff
@@ -0,0 +1,85 @@
+@import "../colors";
+
+.nomination-top-section {
+  h2 {
+    color: $dark-blue;
+  }
+}
+
+.nomination-form {
+  background: $pink;
+  margin: 1rem 0;
+  padding: 1rem;
+  border-radius: 1rem;
+
+  h3 {
+    color: $white;
+    font-weight: 100;
+  }
+
+  select,
+  input {
+    border-radius: 0.2rem;
+  }
+
+  form {
+    margin: 0;
+    padding: 0;
+  }
+}
+
+.nomination {
+  background: $dark-blue;
+  border: 0.2rem solid $white;
+  border-radius: 0.4rem;
+
+  h5 {
+    color: $pink;
+  }
+
+  h6 {
+    height: 2.8rem;
+    overflow-y: scroll;
+    color: $white;
+  }
+
+  .button-group {
+    padding: 0 0.5rem;
+    height: 2.8rem;
+  }
+
+  .button-pink {
+    margin: 0;
+  }
+
+  ::-webkit-scrollbar {
+    background: transparent;
+  }
+
+  ::-webkit-scrollbar-track {
+    background: $dark-blue;
+  }
+}
+
+.ribbon {
+  position: absolute;
+
+  img {
+    height: 3.5rem;
+  }
+
+  span {
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered bottom, color, position, right